substrate/frame/election-provider-multi-block/src/signed/tests.rs

```diff
@@ -324,10 +679,190 @@ mod solution_data_provider {
 	}
 
 	#[test]
-	fn get_page_works() {
+	fn get_score_returns_entry_with_highest_sum_stake_given_entries_with_equal_minimal_stakes() {
 		ExtBuilder::default().build_and_execute(|| {
 			roll_to_phase(Phase::Signed);
+
 			assert_eq!(<SignedPallet as SolutionDataProvider>::get_score(), None);
+
+			let higher_score =
+				ElectionScore { minimal_stake: 40, sum_stake: 10, sum_stake_squared: 0 };
+			assert_ok!(SignedPallet::register(RuntimeOrigin::signed(40), higher_score));
+
+			let score = ElectionScore { minimal_stake: 40, sum_stake: 5, sum_stake_squared: 0 };
+			assert_ok!(SignedPallet::register(RuntimeOrigin::signed(30), score));
+
+			assert_eq!(<SignedPallet as SolutionDataProvider>::get_score(), Some(higher_score));
+		})
+	}
+
+	#[test]
+	fn get_score_returns_entry_with_lowest_sum_stake_squared_given_entries_with_equal_minimal_stakes_and_sum_stakes(
```

Comment:
I would probably simplify the test names :) this is too long and too descriptive. Again I think this test could be merged in the above test, where all the permutations of scores and ordering are tested.

Reply:
Yeah, I got carried away with these names a little :)
I am very much for desciptive but true that this is too much. 
I'll work on it.